### DISCUSSION TITLE
Support .NET Interactive / Polyglot Notebooks

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#5776] [Feature Requests]
### Description

This seems like a promising approach for a multilingual notebook.

https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode
https://github.com/dotnet/interactive/blob/main/docs/FAQ.md

**Comment by jon-durand** [comment]
It would be great to be able to integrate this and use .NET languages with Quarto!
As far as I've heard Microsoft is slowly trying to push F# as a competitor with Python in the ML/Data Science space.
